virtualizarr/zarr.py

```diff
@@ -96,3 +106,67 @@ def ceildiv(a: int, b: int) -> int:
     See https://stackoverflow.com/questions/14822184/is-there-a-ceiling-equivalent-of-operator-in-python
     """
     return -(a // -b)
+
+
+def dataset_to_zarr(ds: xr.Dataset, storepath: str) -> None:
+    """
+    Write an xarray dataset whose variables wrap ManifestArrays to a Zarr store, writing chunk references into manifest.json files.
+
+    Not very useful until some implementation of a Zarr reader can actually read these manifest.json files.
+    See https://github.com/zarr-developers/zarr-specs/issues/287
+    """
+
+    from virtualizarr.manifests import ManifestArray
+
+    _storepath = Path(storepath)
+    Path.mkdir(_storepath, exist_ok=False)
+
+    # TODO should techically loop over groups in a tree but a dataset corresponds to only one group
+    # TODO does this mean we need a group kwarg?
+
+    consolidated_metadata: dict = {"metadata": {}}
```

Comment:
small wrinkle for v3 -- we don't have a consolidated metadata spec extensions yet. No harm in putting this in for now but v3 readers are not going to know what to do here until we formalize this a bit.

Reply:
I think you could also forgo accumulating the consolidated metadata as you do here, and instead, just call `zarr.consolidate_metadata(store)` on the store after you've written all the metadata docs.

Reply:
I think in that case I should probably take out the consolidated metadata part and just make this into a bare-bones implementation of a "chunk manifest ZEP"-compliant store.